test/e2e/command/botkube.go

```diff
@@ -83,6 +83,10 @@ func (c *context) testBotkubeCommand(t *testing.T) {
 				"  - statefulsets\n" +
 				"  - storageclasses\n",
 		},
+		"BotKube notifier showconfig": {
+			command:  "notifier showconfig",
+			expected: "<@U023BECGF> notifier showconfig",
+		},
```

Comment:
@anoopmsivadas Can you please remove this part. This is already being addressed in the separate PR.
The rest looks good to me

Reply:
@PrasadG193 so I have to remove this part and create a new pull request, right?